hooks/pre-exit

```diff
@@ -2,13 +2,17 @@
 
 set -euo pipefail
 
-nvm_installation_dir=$(buildkite-agent meta-data get "automattic-nvm-installation-dir")
-# if file $nvm_dir/.automattic-nvm-dir-marker" exists
-if [[ -f "$nvm_installation_dir/.automattic-nvm-dir-marker" ]]; then
+# Use the same deterministic directory path as in pre-command
+TEMP_DIR="${TMPDIR:-/tmp}"
+nvm_dir="${TEMP_DIR}/automattic-nvm-${BUILDKITE_JOB_ID}"
+
+# Check if the directory exists and has our marker file
```

Comment:
Comment is now incorrect… and given the simplification of the logic, is probably not even that useful anymore
```suggestion
```